features/org.pitest.pitclipse.core.feature/feature.xml

```diff
@@ -228,6 +228,8 @@
    <requires>
       <import plugin="org.eclipse.core.runtime" version="3.11.1" match="compatible"/>
       <import plugin="org.eclipse.jdt.core" version="3.11.2" match="compatible"/>
+      <import plugin="org.pitest.pitclipse.runner" version="2.0.0" match="compatible"/>
```

Comment:
these two are not required in the `<requires>` section since they are included as plugins or transitively through the included features of ours

Reply:
If so, the calculation of the Eclipse PDE is not working properly.

My steps to create this were:
1. Open this file in Eclipse
2. Switch to the dependency tab of the feature.xml
3. Click on compute
![grafik](https://user-images.githubusercontent.com/50097340/128979392-ba339694-fbe8-40eb-a5e3-cd72e43aed1a.png)

Reply:
Probably it finds those dependencies and it adds them but they're useless and probably dangerous. Usually I never used that mechanism because it finds compile dependencies that are implicitly and automatically taken into consideration by P2 itself. This list should be used to specify runtime dependencies that cannot be computed automatically